docker/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM debian:9.6-slim
 
-# Default google-fluentd at the unstable version.
+# Default google-fluentd at the stable version.
 # Use `docker build --build-arg REPO_SUFFIX=<CUSTOM_REPO_SUFFIX>` to build
 # images with a specific version of the package.
```

Comment:
Let's change this comment from "build images with a specific version" to "override the version"...

Reply:
Done.